ccd/deployment.yaml

```diff
@@ -0,0 +1,825 @@
+---
+#
+# NAMESPACE
+#
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ccd
+---
+#
+# IDAM-AUTHENTICATION-WEB CONFIG
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: idam-authentication-web
+  namespace: ccd
+data:
+  IDAM_API_URL: 'http://idam-api'
+  IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_BULK_SCAN_ORCHESTRATOR: '123456'
+  IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_CCD_GATEWAY: '123456'
+  IDAM_CONTINUE_URL_VALIDATOR_ENFORCE_TLS: 'false'
+---
+#
+# IDAM-API CONFIG
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: idam-api
+  namespace: ccd
+data:
+  JAVA_OPTS: '-Xms8m -Xmx256m -Djava.security.egd=file:/dev/./urandom -XX:MaxMetaspaceSize=128m'
+  SPRING_DATASOURCE_URL: 'jdbc:postgresql://shared-database:5432/idam'
+  IDAM_TESTING_SUPPORT_ENABLED: 'true'
+  SPRING_MAIL_HOST: 'smtp-server'
+  NOTIFY_API_KEY: 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'
+  NOTIFY: 'false'
+  NOTIFY_CMC_WELCOME_USER_TEMPLATE: 'fake'
+  NOTIFY_DIVORCE_WELCOME_USER_TEMPLATE: 'fake'
+  NOTIFY_SSCS_WELCOME_USER_TEMPLATE: 'fake'
+  NOTIFY_RESET_PASSWORD_TEMPLATE: 'fake'
+  NOTIFY_PROBATE_ACTIVATE_USER_TEMPLATE: 'fake'
+  NOTIFY_DIVORCE_ACTIVATE_USER_TEMPLATE: 'fake'
+  NOTIFY_SSCS_ACTIVATE_USER_TEMPLATE: 'fake'
+  NOTIFY_CCD_ACTIVATE_USER_TEMPLATE: 'fake'
+  NOTIFY_BAR_ACTIVATE_USER_TEMPLATE: 'fake'
+  NOTIFY_FREG_ACTIVATE_USER_TEMPLATE: 'fake'
+  IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_CMC_CITIZEN: '123456'
+  IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_CMC_LEGAL: '123456'
+  IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_CCD_GATEWAY: '123456'
+  http_proxy: ''
+  https_proxy: ''
+  no_proxy: ''
+  REFORM_ENVIRONMENT: 'test'
+  IDAM_BSP_SYSTEMUPDATE_USER: 'bulkscanorchestrator+systemupdate@gmail.com'
+  IDAM_BSP_SYSTEMUPDATE_PASSWORD: 'Password12'
+  IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_BSP: '123456'
+  IDAM_BSP_WHITELIST: 'https://localhost/receiver'
+---
+#
+# SERVICE-AUTH-PROVIDER-API CONFIG
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: service-auth-provider-api
```

Comment:
We should be able to use the AAT version of s2s and not need to run this.
Unless there's a good reason to bring everything into AKS?